tests/config/driverconfig/powerstore/v2.4.0/controller.yaml

```diff
@@ -0,0 +1,314 @@
+apiVersion: v1
```

Comment:
@panigs7, do we need these files repeated under tests for each of the supported version. Can we use them from driverconfig if required and not have it repeated

Reply:
Discussed offline. Will be taking as an investigation